src/integrationTest/java/uk/gov/hmcts/reform/iataskconfiguration/CamundaCancellationTaskTest.java

```diff
@@ -39,6 +39,16 @@ void test_response_of_cancellation_dmn_table(String fromState, String event, Str
         assertThat(dmnDecisionRuleResults.getFirstResult().getEntryMap(), is(result));
     }
 
+    @DisplayName("Set warn for tasks")
+    @ParameterizedTest
+    @CsvSource({"\"\",makeAnApplication,\"\""})
+    void test_response_of_warn_dmn_table(String fromState, String event, String state) {
+        VariableMap result = new VariableMapImpl();
+        result.putValue("action","Warn");
+        DmnDecisionTableResult dmnDecisionRuleResults = evaluateDmn(fromState,event,state);
+        assertThat(dmnDecisionRuleResults.getFirstResult().getEntryMap(), is(result));
```

Comment:
There's only one WARN action. can we add an assertion to verify the action is "Warn"?